backend/app/generate_badges.py

```diff
@@ -13,7 +13,8 @@ class GenerateBadges:
     def __init__(self):
         self.NUMBER_OF_BADGES_PER_PAGE = 8
         self.APP_ROOT = os.path.dirname(os.path.abspath(__file__))
-        self.UPLOAD_FOLDER = os.path.join(self.APP_ROOT, 'static/uploads')
+        self.UPLOAD_FOLDER = os.path.join(
+        self.APP_ROOT, 'static/badge_backgrounds')
```

Comment:
@igniteeng000 Please keep it in one line only

Reply:
I can't do this python does not support long lines